build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
     classpath 'com.github.jengelman.gradle.plugins:shadow:5.0.0'
     classpath 'com.palantir.baseline:gradle-baseline-java:3.36.2'
     classpath 'com.palantir.gradle.gitversion:gradle-git-version:0.12.3'
-    classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.14.0'
+    classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.20.0'
```

Comment:
maybe worth bumping to the latest version, which is `5.14.0`?

Reply:
Not sure if bumping the major version would cause issue, but let me try 👍